src/js/ui/pokemon.js

```diff
@@ -0,0 +1,160 @@
+import { 
+    capitalizeFirstLetter, 
+    parseToThreeDigits, 
+    convertMetersToFeetAndInches, 
+    convertKgToLb, 
+    objectPropsToString,
+    parseFromSnake 
+} from '../utils.js';
+import { showPokemonUI, appendToMain, appendTags } from '../dom.js';
+
+export function handleNavigation(e) {
+    if (e.target.classList.contains('unselected')) {
+        const $nav = document.getElementById('nav-info');
+        const $selected = $nav.children[$nav.dataset.selected];
+        
+        document.querySelector(`[data-section="${$nav.dataset.selected}"]`).classList.add('hidden')
+        document.querySelector(`[data-section="${e.target.dataset.button}"]`).classList.remove('hidden')
+
+        $selected.classList.remove('selected');
+        $selected.classList.add('unselected');
+
+        e.target.classList.remove('unselected');
+        e.target.classList.add('selected')
+    
+        $nav.dataset.selected = e.target.dataset.button;
+    }
+}
+
+export function showPokemonInfo(ID, basic, breeding) {
+    // const color = breeding.color.name;
+    appendToMain(createSection());
+
+    makeHero(basic);
+    makeAbout(basic);
+    makeBreeding(breeding);
+    makeStats(basic.stats);
+    showPokemonUI();
+}
+
+function makeBreeding(data) {
```

Comment:
The same thing I told you before with list/array works with the data word. Everything is data or information, you have to be more specific with what really means `data`.

Reply:
Understood